src/NoteWorkspace.ts

```diff
@@ -70,8 +70,8 @@ export class NoteWorkspace {
   // This will allow us to potentially expose these as settings.
   // Note for the future: \p{L} is used instead of \w , in order to match to all possible letters
   // rather than just those from the latin alphabet.
-  static _rxTagNoAnchors = '(?<= |,|^)#[\\p{L}\\-_]+'; // used to match tags that appear within lines
-  static _rxTagWithAnchors = '^#[\\p{L}\\-_]+$'; // used to match entire words
```

Comment:
- `_rxTagNoAnchors` is now `_rxTag`
- `_rxTagWithAnchors` was never used
- `_rxBeginTag` now allows matching an empty tag `#` with the allowed preceding chars